socialjusticehackathon/project-template
# Get the template started

## Overview

This provides some instructions and basic files (especially a README, LICENSE, and `.github` files for project leads to have pre-set-up for them so they can dive into coding sooner.



### Does this close any currently open issues?


Closes none

### Testing Instructions

- Read through the README instructions and ensure they make sense and are easily readable
- Same for the README-example
- Follow the instructions in the [usage section](https://github.com/socialjusticehackathon/project-template/blob/main/README.md#usage) to make sure the steps, if followed by rote, would lead the project leads to success and not into ruin
